### ISSUE TITLE
mapstruct-on-gradle: warning: The following options were not recognized by any processor: '[mapstruct.verbose]'

### ISSUE THREAD

**Issue by 62mkv** [mapstruct/mapstruct#2036]
when I update `build.grade` compileJava options definition to look as follows: 

```
tasks.withType(JavaCompile) {
    options.compilerArgs = [
            '-Amapstruct.verbose=true'
    ]
}
```
(all other things being equal)
I get this instead of increased verbosity: 

`warning: The following options were not recognized by any processor: '[mapstruct.verbose]'`

is there anything else that needs to be addressed? Is this even supported option yet?

**Comment by 62mkv** [comment]
or should it be raised against `mapstruct-examples` repo?

**Comment by filiphr** [comment]
It is not working because it would be released as part of 1.4
